index.bs

```diff
@@ -3625,6 +3678,44 @@ as seen for example in [[#example-ws-no-backpressure]].
  1. Return ? [$AcquireWritableStreamDefaultWriter$]([=this=]).
 </div>
 
+<h4 id="ws-transfer">Transfer via postMessage()</h4>
```

Comment:
I'd suggest basically the same editorial tweaks for this section.

Reply:
Done.